elasticdl/python/master/rendezvous_server.py

```diff
@@ -11,20 +11,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from horovod.runner.common.util.hosts import get_host_assignments, parse_hosts
-from horovod.runner.http.http_server import RendezvousServer
+try:
+    from horovod.runner.common.util.hosts import (
+        get_host_assignments,
+        parse_hosts,
+    )
+    from horovod.runner.http.http_server import RendezvousServer
+
+    _HOROVOD_INSTALLED = True
+except ImportError:
+    _HOROVOD_INSTALLED = False
```

Comment:
What behavior do we expect in master and workers if horovod is not installed?

Reply:
When we use `ParameterServerStrategy`,  we don't need to install Horovod. And the master and worker will not call the module about Horovod like `RendezvousServer` and `training_with_allreduce` if the strategy is `ParameterServerStrategy`.

Reply:
What will happen if we use AllReduceStrategy and HOROVOD is not installed?

Reply:
The master will raise errors.